modules/likes/style.css

```diff
@@ -1,17 +1,11 @@
 /**
  * Like Button toolbar button, loading text & container styles
- *
- * @todo: doesn't look like "style.css" is used anymore.
  */
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
```

Comment:
The concatenation process doesn't put this import at the beginning of the file so it doesn't work when `jetpack.css` is being used.